go/protocol/keybase1/extras.go

```diff
@@ -2680,6 +2680,26 @@ func (e TeamInviteMaxUses) IsValid() bool {
 	return e > 0 || e == TeamMaxUsesInfinite
 }
 
+func max(a, b int) int {
+	if a >= b {
+		return a
+	}
+	return b
+}
+
+func (e *TeamInviteMaxUses) String(alreadyUsed int) string {
+	if e != nil && e.IsInfiniteUses() {
+		return "unlimited uses left"
+	}
+	var maxUses int
+	if e == nil {
+		maxUses = 1
+	} else {
+		maxUses = int(*e)
+	}
+	return fmt.Sprintf("%d/%d uses left", max(maxUses-alreadyUsed, 0), maxUses)
```

Comment:
I don't think that's a good format here if someone's seeing this for the first time, but I can't think of anything better.

Maybe just `Joined: %d; Limit: %d`. It's not in our designs because this was not planned for the GUI, but I think it should be since you'll be able to make these invites from cli.

Reply:
/shrug i think it's ok. This is displayed in tab formatted table so I was trying to be concise